packages/components/organisms/f-checkout/src/tenants/en-AU.js

```diff
@@ -128,6 +128,12 @@ const messages = {
                 buttonText: 'Ok'
             },
 
+            GEOLOCATION_REQUIRED: {
+                title: 'Something went wrong',
```

Comment:
Is this the correct title?  _Something went wrong_ sounds a but unfriendly and many people (like my mum) wouldn't read past the title.